### DISCUSSION TITLE
Outputs in Termux bash shell, after installation

### DISCUSSION THREAD

**Discussion by b9Joker108** [akinomyoga/ble.sh#431] [Q&A]
Hello

My current daily driver, login and default shell is zsh, but I often use bash directly. I just installed `ble.sh`in Termux with the quick install method. It all appears to have gone as planned.

The first time executing `bash` after install, I got the following output:

```bash 
❯ bash
ble.sh: insane environment: $USER is empty.
ble.sh: modified USER=u0_a819
ble/term.sh: updating tput cache for TERM=xterm-256color... done
```

The second time executing `bash` after install, I got the following output:

```bash
❯ bash
bash: .sound: No such file or directory
```

It seems that `ble.sh` is looking for a 'sound' dotfile (`.sound`), but couldn't find it. Do I just create it? If so, what should it be populated with?

Any assistance greatly appreciated.

Thanking you in anticipation
Beauford

**Comment by akinomyoga** [comment]
`ble.sh` doesn't use (and has never used) any file named `.sound`. I tried it in Termux with the latest version of ble.sh, but the problem doesn't arise. What is the content of your `~/.bashrc` and `~/.bash_profile` (if any)?

**Comment by b9Joker108** [comment]
SORRY to trouble you, it was my error. In Termux, my daily driver, login and default shell is currently zsh. Well, it has been for the last two years. Prior to zsh, I solely used bash. But, I am experimenting with other shells, and endeavouring to be shell-agnostic, and currently have eighteen different shells installed in my Termux. I am generally confused in life, but this shell complexity is pushing-the-envelope, even for me. 

In a .bashrc I was temporarily trialling, it actually has the code snippet: `source ~/.sound`, which I overlooked. I don't currently have a .sound configuration file, so that was the source of the error, and I noobily and stoopidly, attributed it with the new installation of ble.sh, which I was simultaneously exhilarated by, and in awe of, as well as duly perplexed by. Ble.sh's intricacy and complexity is incredible to me. It truly is a marvellous gift, the code that you and your project's contributors have given.

Sorry, for troubling you with my stupidly. I hope to learn much from your project's code.

**Reply by akinomyoga** [reply]
Oh, OK. It's nice to hear that you've identified the cause. Thank you for trying ble.sh!
